packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/toolbar/design.md

```diff
@@ -0,0 +1,94 @@
+### When to use
+The toolbar should be used when users may need to manage and manipulate a data set, whether it is displayed in a table view, list view, card view, or any other view that requires manipulation and does not already offer controls to do so. There are multiple toolbar variations in order to address a variety of use cases. The toolbar may be further customized by using a unique set of components from the related components list.
+
+### When not to use
+The toolbar should not be used when the data view already includes the necessary controls to manipulate the data set.
+
+### Layout considerations
+![layout](img/toolbar-layout.png)
+
+The toolbar should appear below the page title, if one exists. It should always appear above the data set to which it applies. Toolbars should stretch to the width of the data view to which it applies, with the elements aligned as specified below. When filters are applied, the toolbar will expand in height to make space for a row of filter chips. Upon clearing the applied filters, the toolbar will collapse to it’s default height.
+
+Within the toolbar, layouts may vary depending on the use case. When used, bulk selection should be the left-most element. Filters and global actions may follow. When pagination is used it should be the right-most element. When pagination is not used, the item count should be the right-most element.
+
+## Variations
+**Basic toolbar**
+![basic](img/basic-toolbar.png)
+The basic toolbar is sufficient for most cases. It includes filters, global actions, and item count. The Toolbar is shown here with an Attribute-Value (textbox) filter, but a variety of filter types may be used.
+1. **Item count**: When a filter is applied, the item count will be updated to represent the filtered subset of items. For example, “10 of 17 items.” The item count should be the right-most element.
+
+**Toolbar with pagination**
+![pagination](img/toolbar-pagination.png)
+When pagination is needed, a pagination component can be used.
+1. **Pagination**: When pagination is used, the pagination component replaces the item count. It should be the right-most element.
+
+**Toolbar with filter chips**
+![filterchips](img/toolbar-filter-chips.png)
+When the current settings of the controls are not sufficient to convey applied filters, filter chips should be used.
+1. **Filter chips**: When filters are applied the toolbar will expand and filter chips will be displayed on the second row. Users can clear all filters by clicking the “Clear filters” link, and the toolbar will collapse back into one row.
+
+**Custom toolbar**
+![custom](img/custom-toolbar.png)
+When the current settings of the controls are not sufficient to convey applied filters, filter chips should be used.
+1. **Filter chips**: When filters are applied the toolbar will expand and filter chips will be displayed on the second row. Users can clear all filters by clicking the “Clear filters” link, and the toolbar will collapse back into one row.
```

Comment:
This feels redundant. Maybe it is not needed since it is stated above?

Reply:
Oops, you're absolutely right. This was an accidental repetition. I can replace this with a more accurate description for "Custom toolbar".

Reply:
Agree